axelor-stock/src/main/resources/views/StockMove.xml

```diff
@@ -461,6 +466,8 @@
           readonlyIf="statusSelect &gt; 2" height="3">
           <viewer><![CDATA[<>{toAddressStr}</>]]></viewer>
         </field>
+
+
```

Comment:
Please be careful not to use unnecessary line breaks,  reread your PR to check all your additions and remove unnecessary line breaks